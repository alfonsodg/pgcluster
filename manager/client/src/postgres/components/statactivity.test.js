import React from "react";
import StatActivity from "./statactivity";
import renderer from "react-test-renderer";

test("Statactivty is rendered", () => {
  const rows = [];
  const fetchStatActivity = () => {
      
  }
  const component = renderer.create(
    <StatActivity rows={rows} loading={true} error={null} fetchStatActivity={fetchStatActivity} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
