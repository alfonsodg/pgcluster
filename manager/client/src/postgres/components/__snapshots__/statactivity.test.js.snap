// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Statactivty is rendered 1`] = `
<div>
  <div
    className="row"
  >
    <div
      className="col-md-12"
    >
      <div>
        <button
          className="btn btn-link"
          onClick={[Function]}
        >
          Refresh
        </button>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-md-12"
    >
      <div
        className="loader"
      />
    </div>
  </div>
</div>
`;
